testsuite/MDAnalysisTests/analysis/test_nuclinfo.py

```diff
@@ -174,3 +174,13 @@ def test_hydroxyl(u, seg, i, expected_value):
 def test_pseudo_dihe_baseflip(u, bp1, bp2, i, seg1, seg2, seg3, expected_value):
     val = nuclinfo.pseudo_dihe_baseflip(u, bp1, bp2, i, seg1, seg2, seg3)
     assert_almost_equal(val, expected_value, decimal=3)
+
+
+@pytest.mark.parametrize('b1, b2, seg1, seg2, expected_value', (
+        (2, 3, 'RNAA', 'RNAA', 15.490582),
+        (10, 11, 'RNAA', 'RNAA', 43.71941),
+        (7, 17, 'RNAA', 'RNAA', 168.37481),
+))
+def test_angle_between_base_planes(u, b1, b2, seg1, seg2, expected_value):
+    val = nuclinfo.angle_between_base_planes(u, b1, b2, seg1, seg2)
+    assert_allclose(val, expected_value, rtol=1e-3, atol=0)
```

Comment:
Could you add in tests for the warnings being raised and the two exceptions please?

Reply:
Added.